bpf/lib/lb.h

```diff
@@ -851,7 +852,10 @@ static __always_inline int lb6_local(const void *map, struct __ctx_buff *ctx,
 
 	ipv6_addr_copy(&client_id.client_ip, &tuple->saddr);
 #endif
-
+	if (unlikely(svc->count == 0)) {
+		tuple->flags = flags;
+		return DROP_NO_SERVICE;
+	}
 	/* See lb4_local comments re svc endpoint lookup process */
 	ret = ct_lookup6(map, tuple, ctx, l4_off, CT_SERVICE, state, &monitor);
```

Comment:
No need to restore `tuple->flags` here, as we didn't call `ct_lookup6()` yet:

```suggestion
	if (unlikely(svc->count == 0))
		return DROP_NO_SERVICE;

	/* See lb4_local comments re svc endpoint lookup process */
	ret = ct_lookup6(map, tuple, ctx, l4_off, CT_SERVICE, state, &monitor);
```

Reply:
Sounds good, wasn't sure so I just left it as it was in the goto.